include/deal.II/distributed/shared_tria.h

```diff
@@ -72,6 +72,58 @@ namespace parallel
       typedef typename dealii::Triangulation<dim,spacedim>::cell_iterator        cell_iterator;
 
       /**
+       * Configuration flags for distributed Triangulations to be set in the
+       * constructor. Settings can be combined using bitwise OR.
```

Comment:
maybe say explicitly that `construct_multigrid_hierarchy` can be combined with other options. Otherwise your assert below says they can not be combined arbitrarily.